rhoshift/utils/stability_coordinator.py

```
@@ -105,154 +116,192 @@ def install_operator_with_stability(
-                    'install_time': time.time(),
-                    'health_status': results.get('health_status', HealthStatus.UNKNOWN)
+                    "namespace": namespace,
+                    "install_time": time.time(),
+                    "health_status": results.get("health_status", HealthStatus.UNKNOWN),
                 }
-            
+
             return install_success, results
-            
+
         except Exception as e:
```

Ruff (0.13.1)

Do not catch blind exception: `Exception`

(BLE001)